openxla_patches/f16_abi_clang.diff

```diff
@@ -1,19 +0,0 @@
-upstream CI will fail without this
-diff --git a/xla/service/cpu/runtime_fp16.h b/xla/service/cpu/runtime_fp16.h
-index 3f7af5197..ce4491c5d 100644
---- a/xla/service/cpu/runtime_fp16.h
-+++ b/xla/service/cpu/runtime_fp16.h
-@@ -18,12 +18,7 @@ limitations under the License.
-
- #include <stdint.h>
-
--// _Float16 always gets us the correct ABI type, so use that if available.
--// AArch64 GCC defines __FLT16_MANT_DIG__ even when _Float16 is not available.
--#if defined(__FLT16_MANT_DIG__) && \
--    (defined(__clang__) || !(defined(__GNUC__) && defined(__aarch64__)))
--using XlaF16ABIType = _Float16;
--#elif defined(__x86_64__)
-+#if defined(__x86_64__)
```

Comment:
I remember this one is needed because of upstream CI failure. does openxla already fix this too?

Reply:
This seems related to the arm build. I am rebuilding on my mac laptop to see if any issue pop up.

Reply:
I am having issue compiling the pytorch on M1 in the first step. Not sure if anyone have tried this for XLA before...

Reply:
you can open a pr on pytorch and update the XLA pin in that PR target this pr to verify if you can remove this patch.

Reply:
Opened one here: https://github.com/pytorch/pytorch/pull/132063